app/src/main/java/ch/heigvd/pro/b04/android/Home/EmojiGridAdapter.java

```diff
@@ -82,4 +92,15 @@ private void bindEmoji(Emoji emoji) {
             cardView.setOnClickListener(v -> state.addNewEmoji(emoji));
         }
     }
+
+    private static class InstructionViewHolder extends RecyclerView.ViewHolder {
+        private TextView title;
+        public InstructionViewHolder(@NonNull ViewGroup parent) {
+            super(LayoutInflater.from(parent.getContext())
+                    .inflate(R.layout.activity_instructions, parent, false));
+            title = itemView.findViewById(R.id.instructions);
+
+            title.setText("In order to connect to a poll, please enter its emoji code or scan the given QR code.");
```

Comment:
You probably should set this in the resources (in XML).

Reply:
Done